temporal_sqlalchemy/tests/test_temporal_models.py

```diff
@@ -316,17 +317,16 @@ def test_disallow_flushes_within_clock_ticks_when_strict(self, session, session_
             t.prop_a = 2
 
             with pytest.raises(AssertionError) as excinfo:
-                eval('session.{func_name}()'.format(func_name=session_func_name))
+                getattr(session, session_func_name)()
 
             assert re.match(
                 r'.*flush\(\) has triggered for a changed temporalized property outside of a clock tick.*',
-                str(excinfo)
+                str(excinfo),
```

Comment:
Nitpick:  I usually use `str(excinfo.value)` to get only the exception class rather than the `pytest` `ExceptionInfo` wrapper which sometimes doesn't include the specific message.